### DISCUSSION TITLE
How do I use the GUI ?

### DISCUSSION THREAD

**Discussion by THS-Shiniri** [anidl/multi-downloader-nx#590] [Q&A]
I am on the localhost:3000 and Authenticated for Crunchyroll when searching I get prompted that an ID is needed. Okay cool. How do I get it I see some IDs within Powershell but finding my Show is first of all hard and secondly it still mentions wrong ID.

**Comment by Jaynator495** [comment]
When you search a show on crunchy, you should see the results popup in the GUI, just click one of the results. Then hit "List Episodes" (or type in the box what episodes you want), and add to queue. Hit start and it should download if everything is setup right
